src/main/java/mods/eln/sixnode/ElectricalFuse.kt

```diff
@@ -205,11 +205,7 @@ class ElectricalFuseHolderElement(sixNode: SixNode, side: Direction, descriptor:
     }
 
     override fun onBlockActivated(entityPlayer: EntityPlayer?, side: Direction?, vx: Float, vy: Float, vz: Float): Boolean {
-        if (Utils.isPlayerUsingWrench(entityPlayer)) {
-            front = front.nextClockwise
-            sixNode.reconnect()
-            return true
-        }
+        if (super.onBlockActivated(entityPlayer, side, vx, vy, vz)) return true
```

Comment:
Maybe 
```java
if (onBlockActivatedRotate(entityPlayer)) return true
```
would be more clear when actually reading the code (multiple locations).